langchain/vectorstores/hnswlib.ts

```diff
@@ -153,12 +152,13 @@ export class HNSWLib extends SaveableVectorStore {
       });
       docs.push(newDoc);
     }
-    return HNSWLib.fromDocuments(docs, embeddings);
+    return HNSWLib.fromDocuments(docs, embeddings, docstore);
   }
 
   static async fromDocuments(
     docs: Document[],
-    embeddings: Embeddings
+    embeddings: Embeddings,
+    docstore: Docstore,
```

Comment:
is there anyway to have this be an optional param, and then if its not passed in just initialize with a `new InMemoryDocstore` here?

would make it a bit easier to not have to pass it in everytime

Reply:
This is a great idea, changes incoming.

Reply:
> is there anyway to have this be an optional param, and then if its not passed in just initialize with a `new InMemoryDocstore` here?
> 
> would make it a bit easier to not have to pass it in everytime

Done.